app/controllers/sessions_controller.rb

```diff
@@ -8,6 +9,8 @@ def create
     @user ||= User.create(user_params)
     if @user.valid?
       session[:phone] = @user.phones.first.number
+      sms = SmsSender.new(@user.phones.first.number, @user.phones.first.code,"380680435966", "nananabananananana")
```

Comment:
Move login and password. To rails credentials. [Click](https://www.engineyard.com/blog/rails-encrypted-credentials-on-rails-5.2)